modules/gapi/test/common/gapi_core_tests.hpp

```diff
@@ -142,6 +142,9 @@ struct BackendOutputAllocationTest : TestWithParamBase<>
 struct BackendOutputAllocationLargeSizeWithCorrectSubmatrixTest : BackendOutputAllocationTest {};
 GAPI_TEST_FIXTURE(ReInitOutTest, initNothing, <cv::Size>, 1, out_sz)
 
+GAPI_TEST_FIXTURE(WarpPerspectiveTest, initMatrixRandU, FIXTURE_API(CompareMats, double , double, int, int, cv::Scalar), 6,
+cmpF, angle, scale, flags, border_mode, border_value)
```

Comment:
Please think of people with vertical monitors (like me) or people who split their screens vertically.
This code can be perfectly aligned

Reply:
Done